### PR TITLE
Sheriff

### PR DESCRIPTION
**Pull request by Bigchevrrr**
assignment

### CODE REVIEW

File: assets/chop.jpg
Comment (Sesughter01) [inline assets/chop.jpg:1]: good your image is in the correct folder

File: assets/eat.jpg
Comment (Sesughter01) [inline assets/eat.jpg:1]: ok but choose better quality images

File: assets/facebook.logo.png
Comment (Sesughter01) [inline assets/facebook.logo.png:1]: ok

File: assets/food.jpg
Comment (Sesughter01) [inline assets/food.jpg:1]: choose better quality images

File: assets/football.logo.png
Comment (Sesughter01) [inline assets/football.logo.png:1]: same thing here

File: assets/twitter.logo.png
Comment (Sesughter01) [inline assets/twitter.logo.png:1]: ok

File: assets/vinicom.png
Comment (Sesughter01) [inline assets/vinicom.png:1]: good logo but its positioned wrongly see attached image for correction
sheriff_correctns

**Review by Sesughter01 — COMMENTED** [review]
Please carefully note the corrections and make them

**Review by Sesughter01 — COMMENTED** [review]
make the correctns

**Comment by Bigchevrrr** [comment]
classwork

File: assets/DOM.png
Comment (Sesughter01) [inline assets/DOM.png:1]: Please indicate the html file this DOM is for. Good job

File: class_eight/index.css
Comment (Sesughter01) [inline class_eight/index.css:1]: I will view and revert to you good job

File: class_eight/index.html
Comment (Sesughter01) [inline class_eight/index.html:1]: good

**Review by Sesughter01 — CHANGES_REQUESTED** [review]
Please update your class_two news-homepage project

**Comment by Bigchevrrr** [comment]
pushed sir

**Pull request closed without merging**